src/entities/DealTokenSwap.ts

```diff
@@ -760,6 +761,7 @@ export class DealTokenSwap implements IDeal {
           if (receipt) {
             this.isExecuted = true;
             this.executedAt = new Date((await this.ethereumService.getBlock(receipt.blockNumber)).timestamp * 1000);
+            this.hydrate();
```

Comment:
What specifically needs to be hydrated here?

Reply:
Removed hydrate(). This was used for testing to see why another component wouldn't update and I forgot to remove it. Thanks for catching that!